feedback.md

```diff
@@ -0,0 +1,37 @@
+AMBULO
+======
+<br />
+
+| Team Member | Role                 |
```

Comment:
Be sure you have clear assignments for each role - nice to have the title, but what are your expectations of one another? Be terrific, be specific.